package.json

```diff
@@ -127,6 +121,7 @@
   "dependencies": {
     "json-stable-stringify": "^1.0.1",
     "tslib": "^1.7.1",
+    "typescript-compiler": "^1.4.1-2",
```

Comment:
Why do we need this dependency?

Reply:
Oh hm, this (and all the newline removals) might have been an artifact of my workspace? Thanks for pointing this out! @kanitw would you have any ideas?

Reply:
Try this:
1. `git reset --soft HEAD^`
2. re-add/commit only the relevant files
3. git push origin +hl/example_diverging_stacked_bar (force push, be careful)

Reply:
Perfect, should be good to go. Thanks Will!